tests/src/well_known_types.proto

```diff
@@ -1,8 +1,17 @@
 syntax = "proto3";
 
-import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
+import "google/protobuf/type.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/api.proto";
+import "google/protobuf/descriptor.proto";
 import "google/protobuf/wrappers.proto";
+import "google/protobuf/source_context.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/compiler/plugin.proto";
+import "google/protobuf/duration.proto";
```

Comment:
So I learned that `protoc` bundles all of these includes so you don't actually need to have a protoc_include env var set with these nor do we need to bundle them into the `.crate` for `prost-build`.